.aspect/external_repository_hashes/npm_translate_lock_NDg3NzUwNzA2NQ==

```diff
@@ -0,0 +1,16 @@
+# Input hashes for repository rule: npm_translate_lock(name = "npm", pnpm_lock = "@//:pnpm-lock.yaml")
+//:pnpm-lock.yaml=1507139895
```

Comment:
the format for the hash looks unusual, it's not a sha256 or a resource integrity hash or anything else I'm familiar with

Reply:
yup; hash function which is the only reasonable choice in starlark returns an `int32` as the hash